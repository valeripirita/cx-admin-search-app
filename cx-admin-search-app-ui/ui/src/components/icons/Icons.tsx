import Search from '../icons/search.svg';
import ChevronLeft from '../icons/chevronLeft.svg';
import ChevronRight from '../icons/chevronRight.svg';
import ChevronDown from '../icons/chevronDown.svg';
import Check from '../icons/check.svg';

export {
    Search,
    Check,
    ChevronDown,
    ChevronLeft,
    ChevronRight
}