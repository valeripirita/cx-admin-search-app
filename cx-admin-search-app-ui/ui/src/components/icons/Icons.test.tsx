import React from 'react';

import { afterEach, describe, expect, it } from '@jest/globals';
import { cleanup, render } from '@testing-library/react';

import { Check } from './Icons';

describe('Check', () => {

    afterEach(cleanup)

    it('should render', () => {

        const { asFragment, unmount } = render(<Check />);

        expect(asFragment()).toMatchSnapshot();

        unmount()
    });
})