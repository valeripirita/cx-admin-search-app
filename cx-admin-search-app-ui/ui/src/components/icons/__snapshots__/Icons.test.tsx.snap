// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check should render 1`] = `
<DocumentFragment>
  <svg
    data-file-name="SvgCheck"
  />
</DocumentFragment>
`;
